.bazelrc

```diff
@@ -14,6 +14,16 @@ build:windows --action_env=PATH --define=MINGW_PATH=c:/tools/msys64/mingw64 --de
 # Don't fallback to local toolchains, only use the hermetic ones
 #build --action_env BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
 
+# Caching
+build:ci --config=cache
+common:ci --config=cache
```

Comment:
Theoretically, `build` settings "inherit" `common` settings. Isn't it the case for configs, i.e. doesn't `build:ci` inherit `common:ci`?

Reply:
I didn't even plan to commit these files yet 😅 but I was planning to take @chouquette 's work on https://github.com/DataDog/datadog-agent/tree/chouquette/bazel/remote-caching as a starting point to enable remote caching in a second commit.

As for the comment specifically, I think you're probably right here.

Reply:
To be honest this is copied from https://datadoghq.atlassian.net/wiki/spaces/EEEUSG/pages/2319384594/Bazel+Caching with not much thought put into it besides checking that it works for our purposes :sweat_smile:

Reply:
I've added the lines now "officially" in a separate commit, my intent is that on CI we write to the cache while locally we'll only read.